src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser/newpage--options--.md

```diff
@@ -21,35 +21,35 @@ You can customize the creation of a new page using the following options.
-| viewport                                                            | object  | Sets a viewport size for all pages in the context. `null` disables the default viewport. See: [viewport](#viewport-options) for the options. Defaults to `1280x720`.                                                                                                                                                                                                                     |
+| State | Option                                                              | Type    | Description                                                                                                                                                                                                                                                                                                                                                                              |
+| - | ------------------------------------------------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <span title="Not implemented">❌</span> | acceptDownloads             | boolean | Whether to automatically download files. Defaults to `true`. Follow the progress in [issue #98](https://github.com/grafana/xk6-browser/issues/98). |
+| <span title="Not implemented">❌</span> | baseURL                     | string  | The base URL to use for all relative URLs. For example: `/open-source/` will be converted to `https://k6.io/open-source/` if `baseURL` is set to `https://k6.io`. Follow the progress in [issue #432](https://github.com/grafana/xk6-browser/issues/432). |
+| ✅ | bypassCSP                                                           | boolean | Whether to bypass a page's Content-Security-Policy. |
+| ✅ | colorScheme                                                         | string  | Whether to display a page in dark or light mode by emulating the 'prefers-colors-scheme' media feature. It can be one of `'light'`, `'dark'`, `'no-preference'`. See [page.emulateMedia](/javascript-api/xk6-browser/page#page-emulatemedia-options) for the options and the [example](https://github.com/grafana/xk6-browser/blob/main/examples/colorscheme.js). Defaults to `'light'`. |
+| <span title="Work in progress">🚧</span> | deviceScaleFactor          | number  | Sets the resolution ratio in physical pixels to the resolution in CSS pixels. See the [link](https://github.com/grafana/xk6-browser/blob/main/examples/device_emulation.js) as an example. Defaults to `1`. Follow the progress in [issue #433](https://github.com/grafana/xk6-browser/issues/433). |
```

Comment:
Considering both `newContext()` and `newPage()` receive the same options object, we probably don't want to document it twice.

So we should either link to a central location in both cases, or (preferably) use some templating to import it in both places from a central file. I'm not sure if Gatsby supports this or if we'll have to hack around it with a React component, but we shouldn't repeat documentation like this. It only introduces maintenance issues with keeping both in sync.

Reply:
I've asked for advice on how we can achieve this in the documentation channel (https://loadimpact.slack.com/archives/CCV4TK1DL/p1657554925026749)

Reply:
Created an issue to tackle this another time: https://github.com/grafana/xk6-browser/issues/454